decide/booth/templates/booth/booth.html

```diff
@@ -263,26 +262,30 @@ <h2>[[ voting.question.desc ]]</h2>
                 },
                 decideSend(evt) {
                     evt.preventDefault();
-
-                    // Check if the selected option is 'Voto en Blanco'
-                    // Check if the selected option is 'Voto en Blanco'
-                    // const selectedOption = this.voting.question.options.find(opt => opt.number === this.selected);
-            
-
-                    // if (selectedOption.option === 'Voto En blanco') {
-                    //     this.showAlert("danger", "No puedes votar en blanco");
-                    //     return;
-                    // }
-
-                    var v = this.decideEncrypt(this.selected.toString());
-                                       
-                    var data = {
-                        vote: {a: v.alpha.toString(), b: v.beta.toString()},
-                        voting: this.voting.id,
-                        voter: this.user.id,
-                        token: this.token
+                    if (this.questionType === 'S'){
+                        var v = this.decideEncrypt(this.selected.toString());
+                        var data = {
+                            vote: {a: v.alpha.toString(), b: v.beta.toString()},
+                            voting: this.voting.id,
+                            voter: this.user.id,
+                            token: this.token
+                        }
                     }
-  
+                    else if (this.questionType === 'M') {
+                        var selected = [2,1]
+                        var vote = []
```

Comment:
It seems that these lines have slipped in the code when you were testing your feature. Please take care of it.

Reply:
It should be fixed now.